app/controllers/ncr/work_orders_controller.rb

```diff
@@ -0,0 +1,99 @@
+module Ncr
+  class WorkOrdersController < ApplicationController
+    before_filter :authenticate_user!
+    before_filter :redirect_if_cart_cant_be_edited, only: [:edit, :update]
+
+    def new
+      @work_order = Ncr::WorkOrder.new
+      @approver_email = self.suggested_approver_email
+      @description = ""
+      render 'form'
+    end
+
+    def errors
+      # @TODO we can use a nested model once we get rid of the Cart requirement
+      errors = []
+      if !@description.is_a?(String) || @description.strip().empty?
+        errors = errors << "Description is required"
+      end
+      if !@approver_email.is_a?(String) || @approver_email.strip().empty?
+        errors = errors << "Approver email is required"
+      end
+      if !@work_order.valid?
+        errors = errors + @work_order.errors.full_messages
+      end
+      errors
+    end
```

Comment:
Mind making this a protected method? Just to distinguish that it's a helper method rather than a controller action.